index.html

```diff
@@ -1,4 +1,5 @@
 Roses are shit 
 roses are shit 
-this is a shit poem
-ejaj
\ No newline at end of file
+this is not the same shit poem
```

Comment:
Thank you for this, it looks good but would you mind adding proper periods and commas? It helps us being in line with the code standards we've put up back in 400 AD. 

Thanks! Also for future use. Make sure that your git commits' comments contain a brief description of your code!